### DISCUSSION TITLE
Missing features on Nedit

### DISCUSSION THREAD

**Discussion by johnson329llnl** [Homebrew/discussions#4371] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
Hello Micah>brew config
HOMEBREW_VERSION: 4.0.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: cf6614b50eac1c9f4bf386dc6989f0d443148df8
Last commit: 3 days ago
Core tap: N/A
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.NnnWGN8ZuJ/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Hello Micah>
```

### Output of `brew doctor`

```shell
Hello Micah>brew doctor
==> Downloading https://formulae.brew.sh/api/formula.jws.json
######################################################################## 100.0%
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  pip-numpy

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  brew-pip
Hello Micah>
```

### Description of issue

I installed nedit from Homebrew.  There does not seem to be the usual Alt+keys or shortcuts.  I am attaching a screen-grab for clarity.  You will notice in the image that the usual underscore for File and Edit, etc... are not there.  Additionally, the drop-down menu for Edit does not have the usual Ctrl+z for Undo.  I have been using nedit for over 20 years and have not run in to this issue before.  I would like to keep using nedit. Please advise.

Sincerely, Micah

<img width="959" alt="NEdit_noAltKeys" src="https://user-images.githubusercontent.com/109765062/228976423-a1b010e2-684e-411f-961e-263880ad1e00.png">

**Comment by SMillerDev** [comment]
Did you install it from Homebrew in those 20 years? Or is that a recent development?

**Reply by johnson329llnl** [reply]
Hello SMillerDev,
I first used Homebrew a couple of years ago on a prior MacBook. I believe I installed NEdit then, though I am unsure.   I could have used MacPorts as well, because that is what I used prior to then.  

How are those Alt+keys generated?  Maybe I can look at that, but since I have never seen this, I would not know where to start.

Sincerely,
Micah

**Reply by SMillerDev** [reply]
> How are those Alt+keys generated?

Homebrew just runs `make os` from the upstream source. Maybe you can file an issue upstream for this?

**Comment by hilawe** [comment]
Did you ever find a solution for this issue?
